webpack/dev.config.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable */
+const webpack = require('webpack');
+var webpackMerge = require('webpack-merge');
+var ExtractTextPlugin = require('extract-text-webpack-plugin');
+var commonConfig = require('./common.config.js');
+const IsomorphicPlugin = require('webpack-isomorphic-tools/plugin');
+const path = require('path')
+const webpackIsomorphicToolsPlugin = new IsomorphicPlugin(require('./isomorphic.config'));
+
+module.exports = webpackMerge(commonConfig, {
+  devtool: 'cheap-module-eval-source-map',
+  output: {
+    path: path.resolve('./build'),
+    filename: '[name].js',
+    chunkFilename: '[id].chunk.js'
+  },
+
+  plugins: [
+    new ExtractTextPlugin('[name].css'),
+    new webpack.ProvidePlugin({
+      $: 'jquery',
```

Comment:
@andela-ashuaib @andela-eisaac I think it's not the worst thing to use JQuery in a react app, just don't use anything DOM manipulation. You can use JQuery Ajax and also something like `if ($.isEmptyObject(this.props.currentUser))`.